package.json

```diff
@@ -46,24 +46,24 @@
     "electron-debug": "1.5.0",
     "electron-is-dev": "0.3.0",
     "history": "4.7.2",
-    "immutability-helper": "2.6.6",
+    "immutability-helper": "2.7.0",
     "is-electron": "2.1.0",
-    "lodash": "4.17.5",
+    "lodash": "4.17.10",
     "plist": "3.0.1",
     "prop-types": "15.6.1",
-    "react": "16.2.0",
-    "react-dom": "16.2.0",
+    "react": "16.3.2",
+    "react-dom": "16.3.2",
     "react-redux": "5.0.7",
     "react-router-dom": "4.2.2",
     "react-router-redux": "5.0.0-alpha.9",
     "react-select": "1.2.1",
-    "redux": "3.7.2",
+    "redux": "4.0.0",
```

Comment:
We don't have any code that touched the breaking change for the major version bump here

Reply:
Cool, that was the only one that I was going to ask about :smile:

Reply:
I tried running it on my machine and ran into this issue: https://github.com/reduxjs/redux/issues/2715

I verified it by downgrading to redux 3.7.2 and everything worked fine. After reading [this](https://github.com/zalmoxisus/redux-devtools-extension#12-advanced-store-setup) I added `window === 'object' &&` to [this](https://github.com/tidepool-org/chrome-uploader/blob/master/app/store/configureStore.development.js#L29-L34) which seems to have resolved it. Is that the right way to go? If so, I'll add it to this PR.

Reply:
Hmm, @krystophv , would you mind checking if you can reproduce on macOS? Maybe it's a Linux-specific issue, as I  wasn't able to reproduce on Windows. Also, looks like my `window === 'object' &&` fix breaks the devtools extension.

Reply:
OK, seems that the redux devtools have not yet been updated to officially support redux. Maybe we should hang back on this one for a while?